### ISSUE TITLE
Custom domains list pls

### ISSUE THREAD

**Issue by Traxeronus** [dummylabs/thewatchman#68]
Hi,
thanks for this integration, it is very helpful. I'd like to have an option to include "non standard" domains. F.e. I'm using "Variables" custom integration which creates a "variable.*" domain. Apparently WM ignores such a domains which are not out of the box.
Thanks!

**Comment by dummylabs** [comment]
You're right, so far Watchman uses fixed list of entity domains. There were plans to get actual list of domains upon Home Assistant restart, will look into it.

**Comment by felipecrs** [comment]
@dummylabs I guess this will also be fixed by 0.6.4.

**Comment by dummylabs** [comment]
It would be great if someone could provide an example of custom integration that introduces a custom domain into HA.

**Comment by dummylabs** [comment]
Watchman in its recent version (0.8.3-rc2 announcement: #238) should pick up all custom domains automatically.
